### ISSUE TITLE
Is it correct ? ( getId() method doesn't have return statement ) 

### ISSUE THREAD

**Issue by Kolbasyatin** [gesdinet/JWTRefreshTokenBundle#150]
In the documentation I'v seen this code: 

```
 public function getId()
    {
        $this->id;
    }
```

so, there is a question - why?  Shouldn't I write like this:

```
 public function getId()
    {
       return  $this->id;
    }
```

**Comment by markitosgv** [comment]
@Kolbasyatin where is exactly?

**Comment by Kolbasyatin** [comment]
@markitosgv   at least 
1. in the README file 
2. vendor/gesdinet/jwt-refresh-token-bundle/Document/RefreshToken.php
3. vendor/gesdinet/jwt-refresh-token-bundle/Entity/RefreshToken.php

**Comment by markitosgv** [comment]
Fixed in ff60ce3a585bfdc4eff90d59c9233c0bf404b026
